sktime/forecasting/var_reduce.py

```diff
@@ -0,0 +1,369 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements VARReduce, a VAR-like model combining tabularization with regression."""
+
+__author__ = ["meraldoantonio"]
+
+import numpy as np
+import pandas as pd
+
+from sktime.forecasting.base import BaseForecaster, ForecastingHorizon
+
+
+class VARReduce(BaseForecaster):
+    """
+    A flexible VAR-like forecaster that combines tabularization with regression.
+
+    Fitting proceeds in two steps:
+    1. Tabularization: For each time step and each time series within the
+       multivariate input data, lagged values `X` are generated.
```

Comment:
minor remark: a short mathematical or pictorial description would be useful to the reader

Reply:
Agreed; I added pictorial description in the docstring